src/org/parosproxy/paros/core/scanner/Alert.java

```diff
@@ -356,7 +361,7 @@ public String toPluginXML(String urls) {
 		StringBuilder sb = new StringBuilder(150); // ZAP: Changed the type to StringBuilder.
 		sb.append("<alertitem>\r\n");
 		sb.append("  <pluginid>").append(pluginId).append("</pluginid>\r\n");
-		sb.append("  <alert>").append(replaceEntity(alert)).append("</alert>\r\n");

```

Comment:
The "alert" element should still be returned, otherwise older tools/code would stop working.